src/local-editor-service.ts

```diff
@@ -55,9 +55,25 @@ export class LocalEditorService extends EditorService {
     return feature.sourceRange;
   }
 
-  async getTypeaheadCompletionsAtPosition(
-      localPath: string,
-      position: SourcePosition): Promise<TypeaheadCompletion|undefined> {
+  async getReferencesForFeatureAtPosition(localPath: string,
+                                          position: SourcePosition):
+      Promise<SourceRange[]|undefined> {
+    const document = await this._analyzer.analyze(localPath);
+    const location = await this._getLocationResult(document, position);
+    if (!location) {
+      return;
+    }
+    if (location.kind === 'tagName') {
+      return Array
+          .from(
+              document.getById('element-reference', location.element.tagName!))
+          .map(e => e.sourceRange);
+    }
+  }
+
+  async getTypeaheadCompletionsAtPosition(localPath: string,
+                                          position: SourcePosition):
+      Promise<TypeaheadCompletion|undefined> {
```

Comment:
clang-format seems to have touched this, i guess once the PR for formatting the whole project is merged it'll be no problem.